bc/core/tests.py

```diff
@@ -94,7 +96,71 @@ def test_truncate_descriptions(self):
             result, _ = template.format(**test)
             self.assertTrue(
                 len(result) <= template.max_characters,
-                msg="Failed with dict: %s.\n"
-                "%s is longer than %s"
+                msg="Failed with dict: %s.\n%s is longer than %s"
                 % (test, result, template.max_characters),
             )
+
+
+class TextImageTest(TestCase):
```

Comment:
This should probably be a SimpleTestCase. They're faster if you don't need the DB.

Reply:
You're right! I'm going to use SimpleTestCase instead of TestCase.